app/controllers/caffeine/admin/pictures_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Caffeine
+  module Admin
+    class PicturesController < CaffeineController
+
+      def create
+        picture = Caffeine::Picture.create(image: picture_params)
+        render json: {
+          files: [{
+                id: picture.id,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.